src/nvim/mark.c

```diff
@@ -1552,3 +1552,92 @@ void mark_mb_adjustpos(buf_T *buf, pos_T *lp)
     }
   }
 }
+
+
+// Add information about mark 'mname' to list 'l'
+static int add_mark(list_T *l, char *mname, pos_T *pos, int bufnr, char *fname)
+{
+  dict_T *d;
+  list_T *lpos;
+
+  if (pos->lnum <= 0) {
+    return OK;
+  }
+
+  d = tv_dict_alloc();
+  if (d == NULL) {
+    return FAIL;
+  }
+  tv_list_append_dict(l, d);
+
+  lpos = tv_list_alloc(kListLenMayKnow);
```

Comment:
This never returns NULL. I'll stop repeating myself. Check the function attributes next time.

Reply:
should be fixed in 6764086